crankshaft-engine/src/service/runner/backend.rs

```diff
@@ -50,6 +52,7 @@ pub trait Backend: Debug + Send + Sync + 'static {
         &self,
         task: Task,
         started: Option<oneshot::Sender<()>>,
```

Comment:
I think we can now delete this parameter; we should be using the broadcast sender for getting notifications of tasks starting.